esphome/components/ledc/ledc_output.cpp

```diff
@@ -16,6 +16,7 @@ namespace ledc {
 static const char *const TAG = "ledc.output";
 
 #ifdef USE_ESP_IDF
+static const int MAX_RES_BITS = (LEDC_TIMER_BIT_MAX - 1);
```

Comment:
Parentheses are superfluous:
```suggestion
static const int MAX_RES_BITS = LEDC_TIMER_BIT_MAX - 1;
```